lib/features/thread/data/repository/thread_repository_impl.dart

```diff
@@ -268,7 +278,12 @@ class ThreadRepositoryImpl extends ThreadRepository {
   @override
   Stream<EmailsResponse> loadMoreEmails(GetEmailRequest emailRequest) async* {
     final response = await _getAllEmailsWithoutLastEmailId(emailRequest);
```

Comment:
in this `response`, I did not found `notFoundEmailIds` was updated

Reply:
Updated